drivers/hmis/app/models/hmis/ce/match/unit_group_rule_resolver.rb

```diff
@@ -57,7 +57,9 @@ def compose_priority_expression(rules)
     def compose_requirement_expression(rules)
       # All applicable eligibility requirements are combined with AND.
       rules.filter(&:eligibility_requirement?).
-        map(&:expression).
+        map do |rule|
```

Comment:
@gigxz I mentioned this change briefly on our call, the goal is to avoid (pseudocode):
rule1 says `question_1 = "ANSWER A" OR question_1 = "ANSWER B"`
rule2 says `question_3 = "ANSWER C"`

then the final eligibility expression should be (with parentheses), `(question_1 = "ANSWER A" OR question_1 = "ANSWER B") AND (question_3 = "ANSWER C")`